mgnify_backlog/mgnify_handler.py

```diff
@@ -367,14 +367,20 @@ def set_annotation_jobs_failed(self, study, rt_ticket, failed_runs):
 
         jobs.update(status=failed_status)
 
+    def set_assembly_annotation_job_protein_db(self, assembly_accessions, value=True):
+        jobs = AssemblyAnnotationJob.objects.using(self.database).filter(
+            Q(assembly__primary_accession__in=assembly_accessions))
+        jobs.update(protein_db=value)
+
     def get_request_webin(self, rt_ticket):
         return UserRequest.objects.using(self.database).get(rt_ticket=rt_ticket).user.webin_id
 
     def get_annotation_job_status(self, description):
         return AnnotationJobStatus.objects.using(self.database).get(description=description)
 
     def get_annotation_jobs(self, run_or_assembly_accessions=None, study_accessions=None, status_descriptions=None,
-                            priority=None, pipeline_version=None, experiment_types=None, biome_assigned_only=False):
+                            priority=None, pipeline_version=None, experiment_types=None, biome_assigned_only=False,
+                            not_in_protein_db=False):
```

Comment:
IMO a more generic approach for that flag would be:
```
    def get_annotation_jobs(self, run_or_assembly_accessions=None,
                                            study_accessions=None, status_descriptions=None,
                                            priority=None, pipeline_version=None,
                                            experiment_types=None,
                                            biome_assigned_only=False,
                                            in_protein_db=None)
...
if in_protein_db is not None:
    jobs = jobs.filter(Q(assemblyannotationjob__protein_db=in_protein_db))

```
that way we can re-use that in the future in case we want to get only the ones in the protein_db